src/main/java/com/zufar/onlinestore/common/exception/handler/GlobalExceptionHandler.java

```diff
@@ -40,12 +41,19 @@ protected ApiResponse<Void> buildResponse(Exception exception, HttpStatus httpSt
     }
 
     private List<String> collectErrorMessages(Exception exception) {
-        return exception instanceof MethodArgumentNotValidException methodArgumentNotValidException ?
-                methodArgumentNotValidException
-                        .getBindingResult()
-                        .getAllErrors().stream()
-                        .map(DefaultMessageSourceResolvable::getDefaultMessage)
-                        .toList() : List.of(exception.getMessage());
+        if (exception instanceof UserAlreadyRegisteredException userAlreadyRegisteredException) {
```

Comment:
This exception relates to specific security module. Please, create new separate `SecurityExceptionHandler` class and create method:
```
    @ExceptionHandler(UserAlreadyRegisteredException.class)
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    public ApiResponse<Void> handleUserAlreadyRegisteredException(final UserAlreadyRegisteredException  exception) {
```

Reply:
Done